File: sdkwork-backend-sdk-typescript/src/types/plus-video-form.ts
/** Video creation form */
export interface PlusVideoForm {
  /** Video title */
  title?: string;
  /** Video description */
  description?: string;
  /** Video content URL */
  contentUrl: string;
  /** Aspect ratio */
  aspectRatio?: string;
  /** Duration in seconds */
  duration?: number;
  /** Resolution */
  resolution?: string;
  /** Video width in pixels */
  width?: number;
  /** Video height in pixels */
  height?: number;
  /** File size in bytes */
  fileSize?: number;
  /** Video format */
  format?: string;
  /** Thumbnail URL */
  thumbnailUrl?: string;
  /** Video status */
  status: 'DEFAULT' | 'PROCESSING' | 'COMPLETED' | 'FAILED' | 'DELETED';
}

File: sdkwork-backend-sdk-go/types/plus_video_form.go
package types

// Video creation form
type PlusVideoForm struct {
	Title string `json:"title"`
	Description string `json:"description"`
	ContentUrl string `json:"contentUrl"`
	AspectRatio string `json:"aspectRatio"`
	Duration int `json:"duration"`
	Resolution string `json:"resolution"`
	Width int `json:"width"`
	Height int `json:"height"`
	FileSize int `json:"fileSize"`
	Format string `json:"format"`
	ThumbnailUrl string `json:"thumbnailUrl"`
	Status string `json:"status"`
}
